setupTest.js

```diff
@@ -3,3 +3,10 @@
 process.env.CHOKIDAR_USEPOLLING = true;
 
 jest.setTimeout(140000);
+
+global.afterEach(() => {
+  // We add listeners to signals when creating a new Server instance, but have no cleanup method
+  // So ensure they are removed after each run to prevent EventEmitter memory leak warnings
+  process.removeAllListeners("SIGINT");
+  process.removeAllListeners("SIGTERM");
+});
```

Comment:
Please remove it, it was memory leak on our side, it will be fixed in other PR

Reply:
This is removed now.